### ISSUE TITLE
`type seq` and `functions seq` doesn't show the definition

### ISSUE THREAD

**Issue by dag** [fish-shell/fish-shell#756]
```
> type seq
seq is a function with definition
> functions seq
>
```

Looking at `seq.fish` I can imagine why this happens, but maybe at least `type` should be a tad smarter and at least not say "with definition".

**Comment by Profpatsch** [comment]
Same with grep on my system.

**Comment by pickfire** [comment]
Same on my system but not for `type grep`.

**Comment by krader1961** [comment]
This no longer happens; that is, the definition is now displayed.

**Comment by zanchey** [comment]
Indeed, I think fixed by 29b3b6b31e3acbe2b5474db3b7c3841ba1ed17f9.
